Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apt-get update && \
       ca-certificates \
       curl \
       gnupg-agent \
-      software-properties-common && \
+      software-properties-common \
+      postgresql-client && \
```

Comment:
Need this for the entrypoint to run. Removed it in a heavy-handed commit cleaning up Metro app dependencies earlier.